Soccar [Game-Server]/src/nl/soccar/gameserver/util/MapUtilities.java

```diff
@@ -9,16 +9,23 @@
 import org.jbox2d.dynamics.World;
 
 /**
+ * Utilites for the Map.
+ *
  * @author PTS34A
  */
 public final class MapUtilities {
 
     private static final float WALL_WIDTH = 5;
-    private static final float CORNER_SIZE = 15;
 
     private MapUtilities() {
     }
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)